### PR TITLE
Hotsprings and proper Stronghold

### PR DESCRIPTION
**Pull request by ThePainkiller**
## About The Pull Request

- New indoors hot spring on dorms, separate baths for men and women, increased temperature inside and steamy hot water to have a drink on
image
image
image

- Speaking of drinks, introduces a bottle of Sake to vendors and the hotsprings as well
- New grey brick tiles
- Custodians' stronghold is now tiled with grey brick tiles, making it more akin to a castle as intended, also fixes some lightning issues because candles barely lighten up certain rooms
image
image
image
image
image

- New curtains (currently not craftable) for the hot springs

## Changelog
add: Added Sake bottle to vendor
add: Hotspring curtains
add: New brick tiles (broken) for Custodian's Stronghold
add: New construction ritual recipe: Trade 10 stones for 60 restored and pristine brick tiles. Consider it a placeholder until I can fine-tune the spell, or if I manage to make a ritual that just "repairs" every tile in view.
balance: Watchman scope is now made with uranium and borosilicate glass, but gains nightvision
tweak: CSO gets the science perk back. Crafting tab of PIRS moved to robotics perks (Roboticist and RP can craft everything)
tweak: Onestar boss drops its wreckage now on death, currently can just be taken apart for plasteel and refined, might plan to add more stuff to salvage out of it
fix: Onestar food vendor on the bar is now wired to the Skylight account
refactor: Corrects massive misspellling of "Simulacrum" across a myriad of files
map: Added an indoors hotsprings to dorms area. Custodian's Stronghold has grey bricks now for a more rustic castle look
map: Added spawners for syndicate locker loot chance, rebalances contents of syndicate lockers
/:cl:

### CODE REVIEW

**Comment by ThePainkiller** [comment]
There's bound to be many mapping errors due to the fact this was done way before my PR for areas was merged. I'll try and see how I can fix this...

**Comment by ThePainkiller** [comment]
If anyone asks why the stronghold has "broken tiles" across its floor, it has been stated by Nayu that this is _intentional_, as to signify "the decay" of the values the Custodians want to bring back into this world.
If, for whatever reason, this changes, or players prefer a pristine version of brick tiles, a new subtype can always be created and replaced.

File: code/game/machinery/vendor/tool_vendors.dm [hunk 1]
```diff
@@ -7,14 +7,20 @@
...
-		/obj/item/device/assembly/prox_sensor = 10,
-		/obj/item/device/assembly/igniter = 10,
-		/obj/item/device/assembly/signaler = 10,
+		/obj/item/device/assembly/signaler = 60,
+		/obj/item/device/assembly/prox_sensor = 25,
+		/obj/item/device/assembly/igniter = 25,
+		/obj/item/device/assembly/timer = 25,
+		/obj/item/device/assembly/infra = 25,
+		/obj/item/device/assembly/voice = 25,
+		/obj/item/device/transfer_valve = 25,
```
Comment (Trilbyspaceclone) [inline code/game/machinery/vendor/tool_vendors.dm:16]: this vender is in the hallway what legitmenet use is TTVs to everyday people?

File: code/game/objects/random/closet.dm [hunk 2]
```diff
@@ -105,3 +105,19 @@
...
+
+/obj/random/closet_syndieloot
+	name = "random syndicate locker"
+	icon_state = "closet-red"
+	has_postspawn = TRUE
+
+/obj/random/closet_syndieloot/item_to_spawn()
+	return pickweight(list(/obj/structure/closet/syndicate/suit = 8,
+				/obj/structure/closet/syndicate/personal = 5,
+				/obj/structure/closet/syndicate/nuclear = 3,))
```
Comment (Trilbyspaceclone) [inline code/game/objects/random/closet.dm:117]: this has an uplink with 40 tc, it shouldnt be spawning at all

File: code/game/objects/structures/crates_lockers/closets/syndicate.dm [hunk 3]
```diff
@@ -3,6 +3,8 @@
 	desc = "Why is this here?"
 	icon_state = "syndicate"
 	req_access = list(access_syndicate) //Stop people getting good stuff without access
+	old_chance = -100
+	old_lock_odds = -100
```
Comment (Trilbyspaceclone) [inline code/game/objects/structures/crates_lockers/closets/syndicate.dm:7]:
```suggestion
```

**Comment by Trilbyspaceclone** [comment]
image

File: code/game/machinery/vendor/tool_vendors.dm (hunk 1 above)
Comment (ThePainkiller) [reply code/game/machinery/vendor/tool_vendors.dm:16]: I have no idea but it has always been thus, if you check the slogans
I can remove the references to bomb making and the TTV's if you'd like, toxins already has its own vendor for the stuff anyways

File: code/game/objects/random/closet.dm (hunk 2 above)
Comment (ThePainkiller) [reply code/game/objects/random/closet.dm:117]: > this has an uplink with 40 tc, it shouldnt be spawning at all

Back in Sojourn, on the spider cave that held the crafting station, a long time ago, **you** placed [a very very low chance of spawning this traitor radio](https://github.com/sojourn-13/sojourn-station/pull/1334), and I can personally confirm it sometimes spawning there, on one of the trash piles near the bluecross spawn on the northwesternmost corner of it.
It is still a very low, weighted chance to spawn, on a low-chance-to-spawn spawner, but if you think it more balanced I can just make it have the normal amount of TC

**Comment by ThePainkiller** [comment]
> image Weres the air alarm?

KNEW I WAS FORGETTING SOMETHING AGGGHGHH, I'll add it right away

File: code/game/objects/structures/crates_lockers/closets/syndicate.dm (hunk 3 above)
Comment (ThePainkiller) [reply code/game/objects/structures/crates_lockers/closets/syndicate.dm:7]: Nope, oldifying contents makes this absolutely not worthy of even being there.
Exploding cells, bad rifle, and greyed out voidsuit on a lucky find?
Rewards have to be good, why are we punishing even spawn luck?

**Comment by Trilbyspaceclone** [comment]
> EDIT: Trilby god damn it it's right there
> 
> image

Thats a fire alarm, not an air alarm

**Comment by Trilbyspaceclone** [comment]
image
no one gets money for this vender, dont add it

File: code/modules/projectiles/guns/mods/mods.dm [hunk 4]
```diff
@@ -692,7 +692,8 @@
 	..()
 	var/datum/component/item_upgrade/I = AddComponent(/datum/component/item_upgrade)
 	I.weapon_upgrades = list(
-		GUN_UPGRADE_OFFSET = 0.9,
+		GUN_UPGRADE_NVISION = TRUE, // So that it's not just a barely better ACOG. Would also not stack with thermals.
```
[the remark this answers is not in the document]
Comment (ThePainkiller) [reply code/modules/projectiles/guns/mods/mods.dm:695]: Good point, I'll update the formula when I get home

**Comment by ThePainkiller** [comment]
> image no one gets money for this vender, dont add it

I could make a children of, say, smoke vender and have it have these products but tied to the Skylight account, would that work or do you prefer it just gone?

**Comment by Trilbyspaceclone** [comment]
> > image no one gets money for this vender, dont add it
> 
> I could make a children of, say, smoke vender and have it have these products but tied to the Skylight account, would that work or do you prefer it just gone?

Just gone, those venders are meant to be found outside the colony as is the money, the food they have its quite op

**Pull request merged**